__tests__/client.test.js

```diff
@@ -1,9 +1,17 @@
 import Enzyme from 'enzyme';
 import React from 'react';
 import Adapter from 'enzyme-adapter-react-16';
+import App from '../src/Components/App';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 test('should have access to DOM', () => {
   expect(typeof document).toBe('object');
 });
+
+describe('<App /> ', () => {
+  it('expects App component to exist', () => {
+    const wrapper = Enzyme.shallow(<App />);
+    expect(wrapper.exists()).toBe(true)
+  })
```

Comment:
Missing semi-colon after:
line 15: expect statement
line 16: end of test

Also consider using "test" instead of "it" for your tests. This is the standard in Enzyme, but they ARE interchangeable.